portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsSubmissions.testcase

```diff
@@ -47,7 +47,7 @@ definition {
 
 		FormFields.enableSwitch(fieldName = "Required Field");
 
-		DataEngine.waitForAutoSave();
+		Form.closeSidebar();
```

Comment:
In the macro `DataEngine.waitForAutoSave();` when you click save or publish, the page is automatically refreshed and a new field can be added.
With the removal of the macro, you have to close the sidebar to add a new field.